test/e2e/specs/all-features/record/related-table-link.spec.js

```diff
@@ -393,6 +393,33 @@ describe('View existing record,', function() {
                         done.fail();
                     });
                 });
+
+                it("Page size annotation on domain table should be respected when adding records through Add button in multi-picker", function(){
+                    var related_associate_table = testParams.related_associate_table;
+                    var addRelatedRecordLink = chaisePage.recordPage.getAddRecordLink(related_associate_table);
+                    var EC = protractor.ExpectedConditions, newTabUrl, foreignKeyInputs;
+                    browser.wait(EC.elementToBeClickable(addRelatedRecordLink), browser.params.defaultTimeout);
+
+                    addRelatedRecordLink.click().then(function(){
+                        chaisePage.waitForElement(chaisePage.recordEditPage.getModalTitle());
+                        return chaisePage.recordEditPage.getModalTitle().getText();
+                    }).then(function (title) {
+                        expect(title).toBe("Choose file", "titlte missmatch.");
+
+                        browser.wait(function () {
+                               return chaisePage.recordsetPage.getModalRows().count().then(function (ct) {
+                                   return (ct == 2);
+                               });
+                           });
+                        return chaisePage.recordsetPage.getModalRows().count();
+                    }).then(function(ct){
+                        expect(ct).toBe(2, "association count missmatch for file domain table.");
+                        return browser.executeScript("return $('.modal-close').click();");
```

Comment:
Change this to:
`chaisePage.recordEditPage.getModalCloseBtn().click();`

We should avoid writing jquery functions when we can and use the `chaisePage` factories when possible so the code is easier to read.